### PR TITLE
Adding EOL warning header

### PR DESCRIPTION
**Pull request by ibuziuk**
Signed-off-by: Ilya Buziuk <ibuziuk@redhat.com>

### What does this PR do?
Adding EOL warning header

### What issues does this PR fix or reference?
N/A

### How have you tested this PR?
PR check CI

### CODE REVIEW

File: dockerfiles/che-fabric8/assets/branding/product.json [hunk 1]
```diff
@@ -8,5 +8,8 @@
     "prefetch": {
       "cheCDN":"/api/cdn-support/paths"
     }
+  },
+  "header": {
+    "warning": "Eclipse Che hosted by Red Hat is going to be shut down on <strong>March 1, 2021 at 00:01 GMT</strong>. A new service based on <a href='https://developers.redhat.com/developer-sandbox#assembly-field-sections-59571'>CodeReady Workspaces</a> is available now. Details about the new service, including signing up for it, can be found <a href='https://developers.redhat.com/developer-sandbox'>here</a>."
```
Comment (sleshchenko) [inline dockerfiles/che-fabric8/assets/branding/product.json:13]: I don't think strong tag is supported
https://github.com/eclipse/che-dashboard/pull/152/files#diff-b54815a3469ad6b44f74516907393c946858d8a0f81a53003c4e893d7eae4246R32

It should be easy to add but nobody asked for that 
https://github.com/eclipse/che-dashboard/pull/152#issuecomment-772885788

Comment (sleshchenko) [reply dockerfiles/che-fabric8/assets/branding/product.json:13]: Maybe we should add everything from https://www.w3schools.com/html/html_formatting.asp

Comment (ibuziuk) [reply dockerfiles/che-fabric8/assets/branding/product.json:13]: right, no worries. this is not a problem.

Comment (ibuziuk) [reply dockerfiles/che-fabric8/assets/branding/product.json]: this is just cosmetics, so I removed the unsupported tag ;-)

**Review by sleshchenko — APPROVED** [review]
LGTM

**Comment by ibuziuk** [comment]
[test]

**Comment by ibuziuk** [comment]
[test]

**Review by ScrewTSW — APPROVED** [review]

**Pull request merged**